core-nonfree/story-api-nonfree/src/main/java/org/lgna/story/SRoom.java

```diff
@@ -71,10 +71,6 @@ protected Texture create() {
     private final String textureKey;
   }
 
```

Comment:
now we will just have to wonder ;)

Reply:
Wonder if it is valid? Undoubtedly.

But of course it is the existence of the test that creates doubt.

If I tell you "It's raining outside.", you will likely grab an umbrella.
If I say "Undoubtedly, it is raining outside.", you are more likely to find a window and check.